course/test/src/isometry_TEST.cc

```diff
@@ -35,24 +36,37 @@ GTEST_TEST(Vector3Test, Vector3Operations) {
   EXPECT_EQ(p[1], 2.);
   EXPECT_EQ(p[2], 3.);
 
+
   std::stringstream ss;
   ss << p;
   EXPECT_EQ(ss.str(), "(x: 1, y: 2, z: 3)");
+  EXPECT_TRUE(Vector3::kUnitX == Vector3(1., 0., 0.));
+
+  EXPECT_TRUE(Vector3::kUnitX != Vector3(1., 1., 0.));
```

Comment:
Better make them an initializer list, and implement equality between vectors and initializer lists.